scripts/run-alphanet-parachain.sh

```diff
@@ -72,12 +81,14 @@ echo "parachain $PARACHAIN_INDEX ($PARACHAIN_ID) - p2p-port: $((PARACHAIN_PORT +
 http-port: $((PARACHAIN_PORT + 10 + 1)), ws-port: $((PARACHAIN_PORT + 10 + 2))"
 
 
-if [ -z "$CHAIN" ]; then
-  CHAIN=$ALPHANET_SPEC_RAW
-fi
+
+ADDRESS=(
+  0x4c5a56ed5a4ff7b09aa86560afd7d383f4831cce
+  0x623c9e50647a049f92090fe55e22cc0509872fb6
+)
```

Comment:
It might be useful to add notes about what these accounts are

Reply:
I'm removing them